AndroidManifest.xml

```diff
@@ -1,10 +1 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.mobsandgeeks.saripaar"
-    android:versionCode="1"
-    android:versionName="1.0">
-
-    <uses-sdk android:minSdkVersion="4" android:targetSdkVersion="15" />
-
-    <application></application>
-
-</manifest>
+<manifest package="com.mobsandgeeks.saripaar"/>
```

Comment:
There was an issue last release with not specifying `minSdkVersion` in libraries causing extra permissions to be added. Is this safe?